pipeline_steps/deploy.groovy

```diff
@@ -55,6 +68,13 @@ def upgrade(String stage_name, String upgrade_script, List env_vars) {
           sh "git reset --hard"
         }
         common.prepareRpcGit()
+        if ( stage_name == "Leapfrog Upgrade" ) {
+          create_test_resource_load(
+            env.LEAPFROG_TEST_SERVERS as Integer,
```

Comment:
@git-harry As soon as leapfrog upgrades use this, intra-series will to. It will be a very fast follow. That said, I don't see that as a reason to hold this back - we can be responsible for changing it to suit more general needs once the leapfrog implementation is done and confirmed to be working.